src/mmw/js/src/core/views.js

```diff
@@ -342,12 +343,14 @@ var MapView = Marionette.ItemView.extend({
     _googleMaps: (window.google ? window.google.maps : null),
 
     initialize: function(options) {
-        var map_controls = settings.get('map_controls');
+        var map_controls = settings.get('map_controls'),
+            self = this;
```

Comment:
[convention] In most cases, the `var self = this` declaration is the first in the function body. Prefer:

```js
var self = this,
    map_controls = settings.get('map_controls');
```